src/Tests/Microsoft.NET.Sdk.BlazorWebAssembly.Tests/BlazorWasmStaticWebAssetsIntegrationTest.cs

```diff
@@ -142,5 +146,103 @@ public void StaticWebAssets_Publish_Hosted_Works()
                 publishPath,
                 intermediateOutputPath);
         }
+
+        [Fact]
+        public void StaticWebAssets_BackCompatibilityBuild_Hosted_Works()
```

Comment:
Could we be more specific about the scenario e.g. 

```suggestion
        public void StaticWebAssets_HostedApp_ReferencingNetStandardLibrary_Works()
```

Similarly for the other thing.